### PR TITLE
II-96 logo cypress test

### PR DESCRIPTION
**Pull request by jinyoungkim927**
[Jira ticket](https://immutable.atlassian.net/jira/software/projects/II/boards/294?selectedIssue=II-94)

E2E test:

- User lands on home / collection page
- User clicks a collection
- User clicks the logo from the collection item page
- User returns to home / collection page

### CODE REVIEW

File: cypress/e2e/collection.spec.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Test for clicking collection item from home page 
+
+describe('Clicking into a collection ', () => {
+it('finds the content "type"', () => {
+    cy.visit("http://localhost:3000");
```
Comment (aerim-yi) [inline cypress/e2e/collection.spec.js:5]:
```suggestion
    cy.visit("/");
```
Here as well

File: cypress/e2e/logoClick.spec.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// Click logo should return to collection page 
+
+describe('Clicking into a collection ', () => {
+    it('finds the content "type"', () => {
+
+        const testString = "Siwonchoi"
+
+        cy.visit("http://localhost:3000");
```
Comment (aerim-yi) [inline cypress/e2e/logoClick.spec.js:8]: Here as well

File: cypress/e2e/logoClick.spec.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    
+        cy.contains(testString).click();
+        cy.contains('Intern X Marketplace').click();
+        
+        cy.url().should('equal', 'http://localhost:3000/')
```
Comment (aerim-yi) [inline cypress/e2e/logoClick.spec.js:13]: Here as well

File: cypress/e2e/collection.spec.js (hunk 1 above)
Comment (aerim-yi) [reply cypress/e2e/collection.spec.js:5]: Actually just confirmed with mentors, we should test in localhost and not on the actual website. So ignore this. 

One suggestion is we could change the base URL in the cypress config file so we don't have to write the full path every time.

File: cypress/e2e/logoClick.spec.js (hunk 2 above)
Comment (aerim-yi) [reply cypress/e2e/logoClick.spec.js:8]: As above

File: cypress/e2e/logoClick.spec.js (hunk 3 above)
Comment (aerim-yi) [reply cypress/e2e/logoClick.spec.js:13]: As above

**Review by nur221282 — APPROVED** [review]

**Review by aerim-yi — APPROVED** [review]

**Pull request merged**